scopes/workspace/workspace/eject-conf.cmd.ts

```diff
@@ -14,12 +14,12 @@ type EjectConfOptionsCLI = {
 
 export default class EjectConfCmd implements Command {
   name = 'eject-conf <pattern>';
-  description = 'eject components configuration (create a `component.json` file)';
+  description = 'eject components configuration (create a `component.json` file) - irreversible';
```

Comment:
well, it is reversible, just annoying to reverse.
also, this is "irreversible" in the context of a given workspace. so if i want i can remove the file and re-checkout a previous version, and done..

i think we should remove this note.

Reply:
not really true either. If i tag and export this version, with all sorts of configs that i added to the component.json, there's no way for me to then get those new configs into the component object.

In terms of versioning and component development this is irreversible as there's no way for future versions to build on top of the latest tag without the component.json file and its configs

Reply:
when you `tag`, the configs gets registered as part of the component version. so you can then just delete the file, and things will be fine.
also, after `export`, if you `bit import` that component elsewhere, you don't get that file alongside, as the config is "in" the component.

when you say "irreversible", what do you mean?

Reply:
one sec, the component.json file isnt imported when you import this component?
That's basically reversing what the user tried to do - why would we do that?
@davidfirst is what itay describes correct?

Reply:
@benjgil , yes, it is correct.

When you import a component, we don't care whether previously on another workspace the user decided to eject the config. 
Once you tagged/snapped, all config changes you made in this component.json are now part of the component config. (same as with the config changes you do in .bitmap)

Reply:
i've made a change to this efit
@itaymendel @davidfirst i'm closing this thread an opening a new for the new 'version' of the description